.github/workflows/golangci.yml

```diff
@@ -17,9 +17,10 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-          go-version: 1.19
+          go-version: 1.20.2
```

Comment:
should this be updated? main should still be on 1.19

Reply:
nope this is just for the lint version